core-test/src/test/scala/org/bitcoins/core/psbt/PSBTTest.scala

```diff
@@ -0,0 +1,20 @@
+package org.bitcoins.core.psbt
+
+import org.bitcoins.testkit.util.BitcoinSUnitTest
+import scodec.bits._
+
+class PSBTTest extends BitcoinSUnitTest {
+
+  it must "successfully combine two PSBTs" in {
```

Comment:
Not saying it has to happen on this PR but it might be nice to get in some property based tests for things like this once we have nice generators for PSBTs and PSBT components